app/components/Admin/Room/RoomDetail.jsx

```diff
@@ -29,28 +99,33 @@ function loadRooms() {
     Room.loadRooms();
 }
 
+
+/**
+ * Gets the values from the form and sets properties on tempRoom object
+ */
 function getFormValues() {
-    const number = +document.getElementById('number').value;
-    const name = document.getElementById('name').value;
-    const description = document.getElementById('description').value;
+
     const active = document.getElementById('active').checked;
-    const min = document.getElementById('min-capacity').value;
-    const max = document.getElementById('max-capacity').value;
-
-    tempRoom.number = number;
-    tempRoom.name = name;
-    tempRoom.description = description;
-    tempRoom.active = active;
-    tempRoom.capacity.min = min;
-    tempRoom.capacity.max = max;
+    tempRoom.active = ActiveInput.value;
+
+    tempRoom.number = RoomNumberInput.value;
```

Comment:
Here there were `document.getElementById('example').value` used to get field values, now the `Field` class has a working .value and it will format the value, so numbers are numbers and checkboxes are booleans.